.github/workflows/flutter.yml

```diff
@@ -3,8 +3,8 @@ name: Flutter CI
 on:
   push:
     branches: [master]
-  pull_request:
-    branches: [master]
+  pull_request_target:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
That's not good. Why?

Reply:
This is the way suggested by GitHub itself. I was testing it on the testing branch.

Reply:
Now Since I have tested it. This PR can be closed.